src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +53,36 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService = (MovieSessionService)
+                injector.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        User bob = new User();
+        bob.setEmail("bob@gmail.com");
+        bob.setPassword("12345");
```

Comment:
The password is being set directly on the `User` object without hashing. Ensure that the password is hashed using `HashUtil.hashPassword` before setting it on the user object. This is crucial for securely storing passwords .